expyfun/stimuli/_tracker.py

```diff
@@ -807,6 +818,12 @@ def respond(self, correct):
         self._trackers.flat[self._current_tracker].respond(correct)
         self._trial_complete = True
         self._response_history = np.append(self._response_history, correct)
+        if self.stopped:
+            self._callback(
+                'dealer_%i_stop' % self._dealer_id, json.dumps(dict(
+                    tracker_position=[int(s) for s in self._tracker_history],
+                    responses=[int(s) for s in self._response_history],
+                    x=[int(s) for s in self._x_history])))
```

Comment:
In that case, should the` x_history` outputs for each tracker also be called` x_history`?